### DISCUSSION TITLE
Bibliography per Chapter

### DISCUSSION THREAD

**Discussion by AnaaCorreia** [joaomlourenco/novathesis#338] [Q&A] [answered]
Hello, 

I'm creating a bibliography per chapter inserting:

\printbibliography[heading=subbibliography, segment=\therefsegment, title={\bibname\ for Chapter~\thechapter}]

in the end of the chapter. 

Everything is ok in overleaf. 

When I downloaded the project to TexStudio, the chapter bibliography is not compiled (I changed the default bibliography tool to Biber).

For instance, when I use \cite{ref1}, it shows ref1 in the text (bold) and the program TextStudio ignores the "cite" command, showing only what is inside it even if ref1 doesn't exist . 

Thanks in advance for your help.

 Ana

**Comment by joaomlourenco** [comment, marked as answer]
If it works in Overleaf and not with TexStudio is because TexStudio is not processing the bibliography properly.

I remember someone with a similar problem and he/she solved it defining `latexmk` as the latex processor. See
https://tex.stackexchange.com/questions/351711/texstudio-with-lualatex-and-latexmk-is-this-possible
but I suggest you use
`latexmk -pdf -silent -latexoption="-synctex=1" %`
